test/integration/dnet/helpers.bash

```diff
@@ -194,3 +215,61 @@ function runc() {
     dnet_exec ${dnet} "ip netns exec c unshare -fmuip --mount-proc chroot \"/scratch/rootfs\" /bin/sh -c \"/bin/mount -t proc proc /proc && ${2}\""
     dnet_exec ${dnet} "umount /var/run/netns/c && rm /var/run/netns/c"
 }
+
+function start_zookeeper() {
+    stop_zookeeper
+    docker run -d \
+	   --name=zookeeper_server \
+	   -p 2181:2181 \
+	   -h zookeeper \
+	   jplock/zookeeper
```

Comment:
Can you use `dnephin/docker-zookeeper:3.4.6` instead? Just to make sure we are aligned with other projects and we test from the same source ;)